README.md

```diff
@@ -1,15 +1,22 @@
+# Disclaimer
+[V8](https://v8.dev/) is a community supported project. Our team at Red Hat maintains currency of the latest V8 releases to be used by Node.js on select PowerPC and S390x platforms. A list of platforms as well as their build toolchain supported by Node.js can be found here:
+https://github.com/nodejs/node/blob/master/BUILDING.md
+
+We do not provide official support on V8 if it is used outside the realm of Node.js. V8 is built and tested using the minimum compiler versions currently supported by Node.js (current versions are detailed in the above link). We do not support any other compiler versions or toolchain.
+
+If you need to report a V8 related issue specific to PowerPC or S390x platforms please open a case using the Red Hat Customer Portal https://access.redhat.com/support/cases/
+
 # Building and testing V8 using Docker
-Docker images for building and testing V8 master code on ppc64le and s390x.
+Docker images are used for building and testing V8 for ppc64le and s390x running Linux.
 
-We have 2 types of images, the `base` images which are used for development and the `test` images which are built on top of the base images and are used to build and test V8 commits on Jenkins automatically.
```

Comment:
Should the `base` image / `test` image be referred to as singular?  images vs image.  Are there multiple base images?

> "base" images which are used for development.

The `base` image provides the base configuration with the necessary development toolchains installed to build V8.   The `test` image, built upon the `base`, is used to clone, build and test V8 commits by the Jenkins CI/CD pipeline.